src/snap-points.ts

```diff
@@ -408,9 +413,24 @@ export const createSnapshotForNetwork = async (network: Network) => {
     new BN(lastSnapTimestamp, "hex")
   );
 
-  const lastPointsThatShouldHaveBeenDistrubuted = snapTimeDifference
-    .mul(POINTS_PER_SECOND)
-    .muln(PROMOTED_POOLS.length);
+  const areActivePositionsInPools = PROMOTED_POOLS.map((pool) => {
+    return {
+      pointsPerSecond: pool.pointsPerSecond,
+      hasActiveEntry: Object.keys(eventsObject).some((key) =>
+        eventsObject[key].active.some(
+          (entry) => entry.event.pool.toString() === pool.address.toString()
```

Comment:
If you are comparing public keys you don't need to cast the to string.
The PublicKey instance has method `.equals`. We can leave current approach, but in future it will be better to user pubkey method